rfc/graphql-schema/additionalTypeDefs/byPage/quickOrder.ts

```diff
@@ -0,0 +1,159 @@
+    from: DateTime
+    to: DateTime
+}
+
+input PurchasedProductsSearchInput {
+    orderId: ID
+    search: String
+    dateRange: PurchasedProductDateRangeFilterInput
+    before: String
+    after: String
+    first: Int
+    last: Int
+}
+
+input ConfiguredProductInput {
+    configuredProductId: ID! # shopping-list/adasd-2132e-dwad2 or purchased/as2d2-dd3gv
+    quantity: Int!
+}
+
+input ProductValidationInput {
+    sku: String!
+}
+
+input BulkProductValidationInput {
+    products: [ProductValidationInput!]! 
+}
+
+type ProductValidationErrorType {
+    lineNumber: Int!
+    sku: String!
+    field: String!
+    message: String! # Can we get a translated string from the backend? Or should we send an error type for the frontend
+}
+
+type BulkProductValidationResult {
+    success: Boolean! # Do we need this ? or assume empty errors means success
+    errors: [ProductValidationErrorType!]!
+}
+
+input AddConfiguredProductsDataInput {
+    configuredProducts: [ConfiguredProductInput!]!
+}
+
+input AddConfiguredProductsToCartInput {
+    cartId: ID! # should we make this optional if its for a new cart?
+    data: AddConfiguredProductsDataInput!
+}
+
+type AddConfiguredProductsToCartError implements Error {
+    message: String!
+}
+
+type AddConfiguredProductsToCartResult {
+    cart: Cart # AddCartLineItemsResult also returns cart
+    errors: [AddConfiguredProductsToCartError!]! # Error only contains message: string.
+}
+
+input AddQuoteLineItemsDataInput {
+    lineItems: [CartLineItemInput!]!
+}
+
+input AddQuoteLineItemsInput {
+    # ignoring quote id, since draft quotes are unique per user
+    data: AddQuoteLineItemsDataInput!
+}
+
+type AddQuoteLineItemsError implements Error {
+    message: String!
+}
+
+type AddQuoteLineItemsResult {
+    quote: Quote # AddCartLineItemsResult returns cart, we should return Quote
+    errors: [AddQuoteLineItemsError!]! # Error only contains message: string.
+}
+
+input AddConfiguredProductsToShoppingListInput {
+    shoppingListId: ID!
+    data: AddConfiguredProductsDataInput!
+}
+
+type AddConfiguredProductsToShoppingListError implements Error {
+    message: String!
+}
+
+type AddConfiguredProductsToShoppingListResult {
+    shoppingList: ShoppingList
+    errors: [AddConfiguredProductsToShoppingListError!]!
+}
+
+input AddConfiguredProductsToQuoteInput {
+    # ignoring quote id, since draft quotes are unique per user
+    data: AddConfiguredProductsDataInput!
+}
+
+type AddConfiguredProductsToQuoteError implements Error {
+    message: String!
+}
+
+type AddConfiguredProductsToQuoteResult {
+    quote: Quote
+    errors: [AddConfiguredProductsToQuoteError!]!
+}
+
+extend type CartMutations {
+    bulkProductValidation(input: BulkProductValidationInput!): BulkProductValidationResult!
+    addConfiguredProducts(input: AddConfiguredProductsToCartInput!): AddConfiguredProductsToCartResult!
+}
+
+type QuoteMutations {
+    addLineItems(input: AddQuoteLineItemsInput!): AddQuoteLineItemsResult!
```

Comment:
Same - per our standup /offline discussion need a `create` mutation that takes in two optional params - `title` and `cartLineItem`.